core/domain/user_services.py

```diff
@@ -23,10 +23,10 @@
 import re
 
 from core.platform import models
-current_user_services = models.Registry.import_current_user_services()
-(user_models,) = models.Registry.import_models([models.NAMES.user])
 import feconf
 import utils
+current_user_services = models.Registry.import_current_user_services()
```

Comment:
Okay, just to double check, is this how it is supposed to look?

from core.platform import models
current_user_services = models.Registry.import_current_user_services()
import feconf
import utils

(user_models,) = models.Registry.import_models([models.NAMES.user])
MAX_USERNAME_LENGTH = 50